xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -39,9 +39,20 @@
         <Parameter Name="allowedCipherSuites" Type="System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;" />
       </Parameters>
       <Docs>
-        <param name="allowedCipherSuites">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="allowedCipherSuites">The collection of cipher suites allowed in this policy for negotiation.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Security.CipherSuitesPolicy" /> class that can be used on a client or server authentication.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+> [!NOTE]
+> Defining a cipher suite policy on <xref:System.Net.Security.SslStream> authentication will prevent the OS of the ability to decide which are the best cipher suites to negotiate with and will need of your attention to manually check and update this code. It is strongly recommended to avoid the use of this property and rely on your constantly updated OS policy.
+
+## Remarks
+
+Unlike the OS underlying SSL/TLS implementation, the order of the <xref:System.Net.Security.TlsCipherSuite> elements in `allowedCipherSuites` doesn't guarantee their priority in the client-server negotiation. You can only initialize an instance of this class if you're using Linux systems with OpenSSL 1.1.1 or higher or macOS .
```

Comment:
```suggestion
Unlike the operating system's underlying SSL/TLS implementation, the order of the <xref:System.Net.Security.TlsCipherSuite> elements in `allowedCipherSuites` doesn't guarantee their priority in client-server negotiation. You can only initialize an instance of this class if you're using Linux systems with OpenSSL 1.1.1 or higher or macOS .
```

Reply:
I am taking your suggestion but I also spotted a space before the final dot.
```suggestion
Unlike the operating system's underlying SSL/TLS implementation, the order of the <xref:System.Net.Security.TlsCipherSuite> elements in `allowedCipherSuites` doesn't guarantee their priority in client-server negotiation. You can only initialize an instance of this class if you're using Linux systems with OpenSSL 1.1.1 or higher or macOS.

```